### DISCUSSION TITLE
Error in playback of <song_code>

### DISCUSSION THREAD

**Discussion by deadshotofficial** [TopiSenpai/LavaSrc#116] [Q&A] [answered]
I'm using lavasrc v4.0.0-beta6.
whenever i play some track via spsearch or dzsearch this error shows up
![image](https://github.com/topi314/LavaSrc/assets/63696670/2acc83aa-da04-4649-ad63-b256c3231233)

Follow-up:
I guess you're right about deezer but what about spotify?

**Comment by topi314** [comment, marked as answer]
Provide the correct Deezer master decryption key
